portals/publisher/source/src/app/components/Apis/Details/Properties/Properties.jsx

```diff
@@ -173,7 +173,7 @@ function Properties(props) {
     const iff = (condition, then, otherwise) => (condition ? then : otherwise);
 
     const keywords = ['provider', 'version', 'context', 'status', 'description',
-        'subcontext', 'doc', 'lcState', 'name', 'tags'];
+        'subcontext', 'doc', 'lcstate', 'name', 'tags'];
```

Comment:
Now, what if the user types `lcState` in capital letters?

I think API additional properties are case insensitive @malinthaprasan @tharindu1st ? If so we need to compare in case insensitive way

Reply:
@tmkasun when the user tries to add **"lcState"**, it will not be allowed.

Reply:
https://github.com/wso2/apim-apps/pull/4 added the validations to check uppercase entries of keywords.